app/models/board.rb

```diff
@@ -59,6 +52,11 @@ def move_posts_to_sandbox
     UpdateModelJob.perform_later(Post.to_s, {board_id: id}, {board_id: 3, section_id: nil})
   end
 
+  def add_creator_to_authors
+    return if coauthor_ids.include?(creator_id)
```

Comment:
If this is an after_create how is this possible?

Reply:
...I just copy-pasted this from the post authors after_create, which _does_ have a reason for that, but uh... theoretically if you did something screwy? I can remove it